src/ai/backend/manager/models/vfolder.py

```diff
@@ -890,6 +891,7 @@ async def _update_source_vfolder() -> None:
     return task_id, target_folder_id.folder_id
 
 
+@preserve_termination_log
```

Comment:
If this is a non-long-lived task, it should not be included `@preserve_termination_log` because it will make the termination log to be on the memory forever.  This will keep all of the call history of this method forever until restarting the manager, and as the user/admin tries to delete vfolders over time, this will become the source of memory leak.